forc/src/ops/forc_init.rs

```diff
@@ -1,97 +1,65 @@
 use crate::cli::InitCommand;
-use crate::utils::{
-    defaults,
-    program_type::{ProgramType, ProgramType::*},
-};
+use crate::utils::{defaults, program_type::ProgramType::*};
 use anyhow::Result;
-use forc_util::{print_light_blue, print_light_green, validate_name};
-use serde::Deserialize;
+use forc_util::{println_green, println_light_blue, validate_name};
 use std::fs;
-use std::path::Path;
+use std::path::{Path, PathBuf};
 use sway_utils::constants;
+use tracing::info;
 
-#[derive(Debug, Deserialize)]
-#[serde(rename_all = "lowercase")]
-enum FileType {
-    File,
-    Dir,
-}
-
-// Dead fields required for deserialization.
-#[allow(dead_code)]
-#[derive(serde::Deserialize, Debug)]
-struct Links {
-    git: String,
-    html: String,
-    #[serde(rename = "self")]
-    cur: String,
-}
-
-// Dead fields required for deserialization.
-#[allow(dead_code)]
-#[derive(serde::Deserialize, Debug)]
-struct ContentResponse {
-    #[serde(rename = "_links")]
-    links: Links,
-    download_url: Option<String>,
-    git_url: String,
-    html_url: String,
-    name: String,
-    path: String,
-    sha: String,
-    size: u64,
-    #[serde(rename = "type")]
-    file_type: FileType,
-    url: String,
+fn print_welcome_message() {
+    let read_the_docs = format!(
+        "Read the Docs:\n- {}\n- {}\n- {}",
+        "Sway Book: https://fuellabs.github.io/sway/latest",
+        "Rust SDK Book: https://fuellabs.github.io/fuels-rs/latest",
+        "TypeScript SDK: https://github.com/FuelLabs/fuels-ts"
+    );
+
+    let join_the_community = format!(
+        "Join the Community:\n- Follow us {}
+- Ask questions in dev-chat on {}",
+        "@SwayLang: https://twitter.com/SwayLang", "Discord: https://discord.com/invite/xfpK4Pe"
+    );
+
+    let report_bugs = format!(
+        "Report Bugs:\n- {}",
+        "Sway Issues: https://github.com/FuelLabs/sway/issues/new"
+    );
+
+    let try_forc = "To compile, use `forc build`, and to run tests use `forc test`";
+
+    info!(
+        "\n{}\n\n----\n\n{}\n\n{}\n\n{}\n\n",
+        try_forc, read_the_docs, join_the_community, report_bugs
+    );
 }
 
-#[allow(dead_code)]
-#[derive(serde::Deserialize)]
-struct GithubRepoResponse {
-    sha: String,
-    url: String,
-    // We only care about the tree here
-    tree: Vec<GithubTree>,
-    truncated: bool,
+pub fn canonicalize(path: &PathBuf) -> Result<String> {
```

Comment:
It looks like this `canonicalize` function is intended to be a wrapper around `Path::canonicalize` but returns a `String` in order to more easily be "print"able.

Instead of introducing this function, consider using the `Path::display` method as necessary, e.g. the following is valid:

```rust
println!("Created {}", path.canonicalize()?.display());
```